codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/OperationGenerator.java

```diff
@@ -33,7 +33,7 @@
 /**
  * Generates a client operation and associated custom shapes.
  */
-final class OperationGenerator implements Runnable {
+public final class OperationGenerator implements Runnable {
```

Comment:
Does this have to be public?

Reply:
Yeah it will be used by the  presign url client to refer to the operation's middleware helper function.